### DISCUSSION TITLE
pages.joomlacustomfields.org

### DISCUSSION THREAD

**Discussion by woluweb** [joomlatools/joomlatools-pages#672] [Show and tell]
For JoomlaDay USA 21 (#jdusa21) I made a presentation named 'Joomla, Custom Fields and external data – powerful overrides made easy with Joomlatools Pages'. In this presentation I show step by step how to play with Joomlatools Pages.

With other words it is a sort of *beginners guide*.

- The presentation: https://slides.woluweb.be/pages/pages.html
- The demo site: https://pages.joomlacustomfields.org/
- The code on the demo site itself: https://pages.joomlacustomfields.org/code
- The code on Github: https://github.com/woluweb/Joomlatools-Pages-Demo

**Comment by johanjanssens** [comment]
Great work @woluweb and big thanks for all the effort documenting Pages.
